model-optimizer/mo/front/kaldi/extractors/batchnorm_component_ext.py

```diff
@@ -51,9 +51,6 @@ def extract(cls, node):
         collect_until_token(pb, b'<TestMode>')
         test_mode = read_binary_bool_token(pb)
 
-        if test_mode is not False:
```

Comment:
How the BatchNorm in test mode is resolved later? Do we have an e2e model with such kind of batchnorm running with correct output results?

Reply:
In the past If this mode was enabled MO just didn't allow to convert it. This parameter itself does not affect conversion or inference. @sadolini told that as soon as she remembers she added this restriction as `ad hoc` for a very special case and it is highly likely that it's not needed anymore. We ended up on agreement, that I'll check this. I've checked.

The model in Kaldi with and without TestMode does not show any difference during inference. I think that on e2e we don't have models with enabled TestMode due to the reason I wrote above. I don't think we have negative tests on e2e with models not able to be converted.